config/config.go

```diff
@@ -23,6 +23,8 @@ type RepoConfig struct {
 	GitHubRepoName  string `yaml:"githubRepoName"`
 	GitHubHost      string `default:"github.com" yaml:"githubHost"`
 
+	Reviewers []string `yaml:"reviewers"`
```

Comment:
This definitely doesn't belong in the repo config, a case could be made for user config, but even that's not really useful imo. 
We should start with only the command line option.

Reply:
Hmm, interesting. So in my use case, it's particularly helpful to have it as part of the repo config. 

For example, there are three of us working on the project, so when I post a new review, I always add the same two people.

Just food for thought. If not that, how about if I add the Env option to the string-slice flag? Something like SPR_DEFAULT_REVIEWERS? That seems like a worse scope, though, IMO.

I'm OK if I just have to add a wrapper in the repo to call 

    git spr u -r alice -r bob

but the .spr.yml seemed like a convenient place for it. LMK

Reply:
On second thought, it for sure doesn't belong in the user config, but a case could be made for repo config, when you have a small team and want to send all pull requests to one person, this still feels like a niche case to me though.

Reply:
> At the end of the day, if all the pull requests get the same reviewer, what's the point of setting the reviewer in the first place?

The only value it adds in my case is notifying the user(s) that their review has been requested for a given commit / stack.

For now, though, I'm just going to use the CLI optional arg.